### ISSUE TITLE
normalize_data problem why image/255 and mask/255 when training

### ISSUE THREAD

**Issue by sdjsngs** [HasnainRaz/FC-DenseNet-TensorFlow#17]
before training,   why should image/255 and mask/255   my mask's  each pixel is an index in [0,classnumber]  if mask/255 they will all be 0  (class:background)   when i use pascal VOC 2012 dataset  
one more thing  why use 30 in parse resize normaliza and shuffle   thanks a lot

**Comment by HasnainRaz** [comment]
You can comment out the normalization function in the dataloader. It was written for binary class segmentation problems where the mask has [0, 255] values.

The 30 is just a number I put in, you can increase the buffer size as you wish. There is not restriction on it.
